jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/providers/ContextProvider.java

```diff
@@ -519,4 +519,11 @@ protected void pathAdded(Path path) throws Exception
         if (isDeployable(path))
             super.pathAdded(path);
     }
+
+    @Override
+    protected void pathChanged(Path path) throws Exception
+    {
+        if (isDeployable(path))
```

Comment:
Note that the comment inside `isDeployable` is wrong.  It says ".war" when it should say ".xml"